client-v2/src/components/FrontsEdit/FrontCollectionsOverview.tsx

```diff
@@ -43,7 +43,7 @@ const Container = styled(ContentContainer)<ContainerProps>`
 `;
 
 const ContainerBody = styled.div`
-  width: 380px;
+  width: 360px;
```

Comment:
This brings the collection overview in line with the form, which means there isn't layout jank when opening/closing the form, as the width of the front container doesn't change.